js/initialize.jsx

```diff
@@ -0,0 +1,147 @@
+'use strict'
+
+var React = require('react/addons')
+var ipc = window.require('remote').require('ipc')
+
+var $ = require('jquery-bf')
+var bs = require('bootstrap')
+
+var Initform = React.createClass({
+  displayName: 'Initform',
+  mixins: [React.addons.LinkedStateMixin],
+
+  getInitialState: function () {
+    return {
+      error: null,
+      directory: '',
+      initState: 'uninitialized',
+      keysize: 2048
+    }
+  },
+
+  componentDidMount: function () {
+    var t = this
+    ipc.on('default-directory', function (dir) {
+      t.setState({directory: dir})
+    })
+
+    ipc.on('initializing', function () {
+      t.setState({initState: 'initializing'})
+    })
+
+    ipc.on('initialization-error', function (err) {
+      t.setState({
+        error: err,
+        initState: 'uninitialized'
+      })
+    })
+
+    ipc.on('initialization-complete', function () {
+      t.setState({initState: 'initialized'})
+    })
+  },
+
+  setKeySize: function (size) {
+    var t = this;
+    return function () {
+      t.setState({keysize: size})
+    }
+  },
+
+  initialize: function () {
+    ipc.emit('initialize', {
+      directory: this.state.directory,
+      keysize: this.state.keysize
+    })
+  },
+
+  render: function () {
+    var t = this
+    var initializing = (t.state.initState === 'initializing')
+    var initialized = (t.state.initState === 'initialized')
+    var uninitialized = (t.state.initState === 'uninitialized')
+
+    var description = 'Before starting, ipfs needs to generate a keypair, and create a directory to live in.'
+
+    var success = 'Your ipfs node was setup successfully, you can now close this window and start using the ipfs tray toolbar.'
```

Comment:
> You can now close this window, and start using the ipfs tray toolbar.

this is platform specific and somewhat unclear.


- we say:
  > Your IPFS Node was setup successfully! You can control your node, and access the Web Console, from this new toolbar menu:
  >
  > < show-platform-specific-screenshot-of-the-menu >

- The dialog has "Close" focused and clearly visible.
- We pop open the Menu (and highlight the tray icon or whatever), so the user can see it.
- Make the node be already running, so the user only has to do something if they want to turn it off (alternatively, we could start with the node _off_ to _train_ the user to turn it on and off, but this may not be necessary).